src/Provider.js

```diff
@@ -1,42 +1,37 @@
-import { Children, Component, createContext, createElement } from "react"
+import React, { createContext, useContext, useRef } from "react"
```

Comment:
Can you please stick to the same style of referencing directly through `React.use...`? 

https://github.com/mobxjs/mobx-react/blob/c0ed4b9dcdbe80c277bd03626a89911e2c71fcdb/src/inject.js#L13

Reply:
Oh, sure! Miss it.